acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/step_definitions/PassportAPIStepDefs.java

```diff
@@ -70,10 +70,10 @@ public void passport_user_requests_access_token(String CRIId)
         postRequestToAccessTokenEndpoint(CRIId);
     }
 
-    @And("User sends a new POST request to (.*) endpoint$")
-    public void user_requests_new_access_token(String endpoint)
+    @And("User sends a basic POST request to public (.*) endpoint they get a forbidden error$")
```

Comment:
Can we include without api key in the BDD description 
User sends a basic POST request to public (.*) endpoint without API key and they get a forbidden error